test/powershell/Language/Classes/Scripting.Classes.RunPath.Tests.ps1

```diff
@@ -0,0 +1,42 @@
+Describe "Script with a class definition run path" -Tags "CI" {
+
+    It "Script with a class definition can run from a path without a comma" {
+
+        $FilePath = '.\MyTest.ps1'
+
+        try {
+            'class MyClass { [string]$MyProperty }; $True' | Out-File -FilePath $FilePath
+            $Success = . $FilePath
+        }
+
+        catch {
+            $Success = $False
+        }
+
+        finally {
+            Remove-Item -Path $FilePath
+        }
+
+        $Success | Should Be $True
```

Comment:
Avoid `Should Be $True` - this results in a typically unhelpful message in the log.

I suggest a test that actually uses the class, so maybe something like:

Instead, try something like:

```powershell
`@
class MyClass { static [string]$MyProperty = $PSScriptRoot }
[MyClass]::MyProperty
'@ > $testDrive/My,Test.ps1

& $testDrive/My,Test.ps1 | Should Match ".*My,Test.ps1"
```

Or something like that.